examples/menubar/menubar-2/menubar-2.html

```diff
@@ -136,8 +122,9 @@ <h2 id="ex1_label">Example</h2>
           Gettysburg Address</a>
       </p>
       <script type="text/javascript">
-          var menubar = new MenubarAction(document.getElementById('menubar1'));
-          menubar.init();
+          var menubar      = new MenubarAction(document.getElementById('menubar1'));
```

Comment:
@jongund, In most of our examples, we are following the practice of keeping the js in js files. 
Seems to me that it would be better if we were consistent about this.
Thoughts @jnurthen?
This could be in menubarAction.js or in a small menubar.js.